Workbooks/Workloads/SQL Azure Insights/SQL Azure Insights.workbook

```diff
@@ -2219,10 +2218,10 @@
             "type": 3,
             "content": {
               "version": "KqlItem/1.0",
-              "query": "let data = InsightsMetrics\r\n| where Computer in~ ({OnboardedComputerNames})\r\n| where Namespace contains 'sqlserver_azuredb_waitstats'\r\n| extend Tags = todynamic(Tags)\r\n| extend Database = tostring(Tags[\"database_name'\"]), Instance = tostring(Tags.sql_instance), WaitType = tostring(Tags.wait_type);\r\ndata\r\n| summarize arg_max(TimeGenerated, Val) by WaitType, Database, Instance\r\n| extend Val = iff(TimeGenerated < ({TimeRange:end} - 15m), real(null), Val)\r\n| project Database = iff(Database == Instance, Database, strcat(Database, ' on ', Instance)), WaitType, Size = Val\r\n| extend jkey = strcat(Database, '/', WaitType)\r\n| join kind = inner (data\r\n    | make-series Trend = sum(Val) default = 0 on TimeGenerated from {TimeRange:start} to {TimeRange:end} step {TimeRange:grain} by WaitType, Instance, Database\r\n    | project Database = iff(Database == Instance, Database, strcat(Database, ' on ', Instance)), WaitType, Trend\r\n    | extend jkey = strcat(Database, '/', WaitType)\r\n    ) on jkey\r\n| project Database, WaitType, Size, ['Size trend'] = Trend\r\n\r\n",
+              "query": "let data = InsightsMetrics\r\n| where Computer in~ ({OnboardedComputerNames})\r\n| where Namespace in~ ('sqlserver_azuredb_waitstats', 'sqlserver_waitstats') and Name =~ 'wait_time_ms'\r\n| extend Tags = todynamic(Tags)\r\n| extend Database = tostring(Tags[\"database_name'\"]), Instance = tostring(Tags.sql_instance), WaitType = tostring(Tags.wait_type)\r\n| order by Computer asc, Instance asc, Database asc, Namespace asc, Name asc, WaitType asc, TimeGenerated asc\r\n| extend Val = iff(Computer == prev(Computer) and Instance == prev(Instance) and Database == prev(Database) and Namespace == prev(Namespace) and Name == prev(Name) and WaitType == prev(WaitType) and Val > prev(Val), Val - prev(Val), real(null));\r\ndata\r\n| summarize Val = avg(Val) by WaitType, Database, Instance\r\n| extend Val = iff(isnan(Val), real(null), Val)\r\n| project Database = strcat(Instance, '/', Database), WaitType, Size = Val\r\n| extend jkey = strcat(Database, '/', WaitType)\r\n| join kind = inner (data\r\n    | make-series Trend = avg(Val) default = 0 on TimeGenerated from {TimeRange:start} to {TimeRange:end} step {TimeRange:grain} by WaitType, Instance, Database\r\n    | project Database = strcat(Instance, '/', Database), WaitType, Trend\r\n    | extend jkey = strcat(Database, '/', WaitType)\r\n    ) on jkey\r\n| project Database, WaitType, Size, ['Size trend'] = Trend\r\n\r\n",

```

Comment:
FYI - I did see bugs in some of other workbooks due to setting value to null instead of 0, just pointing out null not always correct if no changes, sometimes 0 is correct.

Reply:
Thanks Doug - good point.

Zero value here is not correct. I believe NaN comes from there being only one entry for the wait stat in the query time range, causing us to now know what the delta is from the previous value. Setting it as null here allows us to represent the cell value as '-' instead of a value. I considered adding a '*' with a legend to indicate that the value changed but we don't know by how much. Choose this  approach to reduce cognitive load.